src/transformers/models/wav2vec2/configuration_wav2vec2.py

```diff
@@ -92,6 +92,33 @@ class Wav2Vec2Config(PretrainedConfig):
             Whether do apply `stable` layer norm architecture of the Transformer encoder. ``do_stable_layer_norm is
             True`` corresponds to applying layer norm before the attention layer, whereas ``do_stable_layer_norm is
             False`` corresponds to applying layer norm after the attention layer.
+        freeze_feat_extract_train (:obj:`bool`, `optional`, defaults to :obj:`True`):
+            Whether to freeze the weights of the feature extractor when training.
```

Comment:
Not sure if this one should be in the model config or a specific training argument. No strong opinion if you prefer it there though.